Cargo.toml

```diff
@@ -83,7 +86,9 @@ version = "0.5.1"
 fs_extra = "1.0.0"
 
 [features]
-beta = []
-default = []
+jeta = []
+default = ["self-update"]
+self-update = ["self_update"]
```

Comment:
I think just make it

```
default = ["self_update"]
self_update = []
```

Then use `self_update` in all the cfg attributes.

Let me know if I'm wrong about this (I totally could be!)
```

Reply:
```
error: failed to parse manifest at `/home/rsmet/src/artifact/Cargo.toml`

Caused by:
  Features and dependencies cannot have the same name: `self_update`
```

It seems to work without the `self_update = []` though.